app/helpers/form_page_error_helper.rb

```diff
@@ -4,6 +4,16 @@ def remove_other_page_errors(lettings_log, page)
     other_page_error_ids.each { |id| lettings_log.errors.delete(id) }
   end
 
+  def remove_duplicate_page_errors(lettings_log)
+    lettings_log.errors.map(&:message)
```

Comment:
Is the map on line 8 needed? Doesn't seem to contribute to the functionality of the method. Group_by handles removing the duplicates. Tested both examples given in your description and it works the same without line 8.

Reply:
Oh, yeah, it doesn't look like it's doing anything, must have accidentally left it! Thanks ,it should be removed now!